src/PhpSpreadsheet/Reader/Html.php

```diff
@@ -699,7 +699,26 @@ private function applyInlineStyle(&$sheet, $row, $column, $attributeArray)
             return;
         }
 
-        $cellStyle = $sheet->getStyle($column . $row);
+        if (isset($attributeArray['rowspan'], $attributeArray['colspan'])) {
+            $columnTo = $column;
+            for ($i = 0; $i < (int) $attributeArray['colspan'] - 1; ++$i) {
```

Comment:
Is the loop necessary? Or can the adjustment of the $columnTo value be done with simple mathematics?

Reply:
Yes Because it is incrementing the column which is A-Z and this is also used in the processDomElement on line number 504 so that is why i have used this. @MarkBaker. we can create separate method now as it is used total 4 times in the whole file.